esi_leap/common/policy.py

```diff
@@ -75,17 +88,21 @@
     policy.DocumentedRuleDefault(
         'esi_leap:offer:get',
         'rule:is_admin or rule:is_owner or rule:is_lessee',
```

Comment:
If the `is_offer_lessee` rule existed I could see it being useful here. (example: owner specifies a project id that does not have the lessee role as a lessee for a newly created offer)